### PR TITLE
Workers: test where Worker/SharedWorker are exposed

### PR DESCRIPTION
**Pull request by annevk**
And remove broken tests.

For https://github.com/whatwg/html/issues/5497.

### CODE REVIEW

File: workers/semantics/multiple-workers/exposure.any.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// META: global=window,worker
+
+test(() => {
+  // Use onfetch as service worker detector
```
Comment (foolip) [inline workers/semantics/multiple-workers/exposure.any.js:4]: How about just `globalThis.ServiceWorkerGlobalScope`?

**Review by foolip — APPROVED** [review]

**Comment by annevk** [comment]
If I run https://web-platform.test:8443/workers/semantics/multiple-workers/exposure.any.sharedworker.html locally in Chrome Canary I get a harness error and basically no useful information. Not sure what's going on.

**Comment by foolip** [comment]
https://wpt.fyi/results/workers/semantics/multiple-workers/exposure.any.sharedworker.html?label=pr_head&max-count=1&pr=23381 looks OK, does it work with `./wpt run --channel dev chrome` locally?

**Comment by annevk** [comment]
Not sure, not familiar with that way of running tests.

**Pull request merged**